packages/common/core/src/form/interfaces/attributes.ts

```diff
@@ -0,0 +1,71 @@
+import type { ODS_COMMON_FIELD_SIZE } from '../constants/ods-common-field-size';
+import type { ODS_INPUT_TYPE } from '../constants/ods-input-type';
```

Comment:
should be renamed `ODS_COMMON_INPUT_TYPE` also

Reply:
Done